M2/Macaulay2/m2/programs.m2

```diff
@@ -89,18 +89,21 @@ findProgram(String, List) := opts -> (name, cmds) -> (
 	instance(opts.MinimumVersion, Sequence) and
 	class \ opts.MinimumVersion === (String, String)) then
 	error "expected MinimumVersion to be a sequence of two strings";
-    pathsToTry := fixPath \ join(
+    pathsToTry := fixPath \ nonnull splice {
```

Comment:
Why not use `flatten`, then you don't need the couple of `toSequence` calls below.

Reply:
Oh yeah -- of course!  Didn't occur to me